src/static/images/doodles-hero.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="1189" height="575" viewBox="0 0 1189 575" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="logo--title" role="img">
+  <title id="logo--title">Teacher Moments</title>
```

Comment:
Ooops this was my fault. Can you move the aria-labelledby, role and the whole title line to the `logo.njk` file?